src/utils/entity-utils.ts

```diff
@@ -12,22 +12,43 @@ export const matchEntitiesByParams = (params: Record<string, string>, paramMatch
   }) : alwaysTrueFn;
 };
 
-export const matchEntitiesByBodyFilters = (body: Request['body'], filterMatch: Optional<Record<string, string>>) => {
-  return filterMatch ? (entity: any) => Object.entries(body).every(([param, value]) => {
-    const entityKey = filterMatch[param];
+export const matchEntitiesByBodyFilters = (body: Request['body'], bodyMatch: Optional<Record<string, string>>) => {
+  return bodyMatch ? (entity: any) => Object.entries(bodyMatch).every(([bodyKey, entityKey]) => {
```

Comment:
did you think about moving the ternary to be inside the function? I think in that you would not need `alwaysTrueFn`.